docs/api/actions.rst

```diff
@@ -24,11 +24,30 @@ Debug
 .. autoclass:: Debug
     :members:
 
+Either
+------
+
+**Aliases**: ``Attempt``,
+``AttemptTo``,
+``GoFor``,
+``Try``
+``TryTo``,
+``Attempts``,
+``AttemptsTo``,
+``GoesFor``,
+``Tries``,
+``TriesTo``
+
+.. autoclass:: Either
+    :members:
+    :exclude-members: except_, else_, otherwise, alternatively, failing_that
```

Comment:
For my knowledge; what does this do?

Reply:
We _**don't**_ want that?

Reply:
These methods are the aliases of the original method, so it will just say the same thing N times, which is very confusing to read in my opinion. We document the aliases in the docstring of the original, so they will show. Here's what it looks like now:
<img width="660" alt="Screenshot 2024-02-22 at 1 15 28 AM" src="https://github.com/ScreenPyHQ/screenpy/assets/27103725/b0e7cd28-fe9f-4479-9162-0050d5ffff29">
here's what it looks like after the PR:
<img width="797" alt="Screenshot 2024-02-22 at 1 16 10 AM" src="https://github.com/ScreenPyHQ/screenpy/assets/27103725/9f28d63b-a062-4aae-90fe-934e68f29004">